docs/dev-env.md

```diff
@@ -46,4 +46,4 @@ Deploying locally
 
 ```
 mkdocs serve
-```
\ No newline at end of file
+```
```

Comment:
What's happening in this line? `/n` got added?

Reply:
I updated the doc with Github's editor and didn't touch that line so I presume it got changed automatically.